client/src/components/layout/ShowTable.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import PlayerTile from "./PlayerTile.js"
+import StatTile from "./StatTile.js"
+
+const ShowTable = ({ user }) => {
+  const [table, setTable] = useState({
+    title: "",
+    notes: "",
+    stats: [],
+  })
+
+  const { tableId } = useParams()
+  const selectedStats = ["pts", "ast", "reb"]
+
+  const getTable = async () => {
+    try {
+      const response = await fetch(`/api/v1/tables/${tableId}`)
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      }
+      const responseBody = await response.json()
+      setTable(responseBody.table)
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getTable()
+  }, [])
+
+  const playerTiles = table.stats.map((stat) => {
+    let player = stat.player
+    player.stats = {
```

Comment:
consider setting up this data structure for `player.stats` instead of `table.stats` on the backend